recipes/spdlog/all/conanfile.py

```diff
@@ -73,6 +73,8 @@ def _configure_cmake(self):
         self._cmake.definitions["SPDLOG_WCHAR_FILENAMES"] = self.options.wchar_filenames
         self._cmake.definitions["SPDLOG_INSTALL"] = True
         self._cmake.definitions["SPDLOG_NO_EXCEPTIONS"] = self.options.no_exceptions
+        if self.settings.os == "iOS":
```

Comment:
maybe watchOS and tvOS as well?

Reply:
I don't have watchOS and tvOS build enviroment.

Reply:
Mostly cases they have the same behavior when building. I also agree adding both.